packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -342,13 +354,21 @@ const tapCopyAboveElement = async ( driver, element ) => {
 
 // Press "Paste" in floating context menu.
 const tapPasteAboveElement = async ( driver, element ) => {
-	const location = await element.getLocation();
-	const action = await new wd.TouchAction( driver );
-	action.wait( 2000 );
-	action.press( { x: location.x + 100, y: location.y - 50 } );
-	action.wait( 2000 );
-	action.release();
-	await action.perform();
+	await longPressMiddleOfElement( driver, element );
+
+	if ( isAndroid() ) {
```

Comment:
`tapPasteAboveElement` was only being used on Android, so I added support for iOS.